.env

```diff
@@ -0,0 +1,10 @@
+MYSQL_ROOT_PASSWORD=rootpassword
```

Comment:
env file still there, let's use `git rm --cached .env` to remove it remotely but keep it locally